### DISCUSSION TITLE
error：size of fp_pp_files should be the same as the size of type_map

### DISCUSSION THREAD

**Discussion by chb-deep** [deepmodeling/dpgen#1327] [Q&A]
Is this error related to POTCAR?
My POTCAR was generated based on this command, and there was no error when generating the training set.
cat  POTCAR_1  POTCAR_2   …… POTCAR_n  >> POTCAR

**Comment by wanghan-iapcm** [comment]
both `fp_pp_files` and `type_map` are lists, the lists should be of the same size
